client/src/components/session/withAuthentication.js

```diff
@@ -1,25 +1,31 @@
-import React from "react";
-import AuthUserContext from "./Context";
-import { withFirebase } from "../firebase";
+import React from 'react';
+import AuthUserContext from './Context';
+import { withFirebase } from '../firebase';
+import axios from 'axios';
 
 const withAuthentication = Component => {
   class withAuthentication extends React.Component {
     constructor(props) {
       super(props);
       this.state = {
-        authUser: JSON.parse(localStorage.getItem("authUser"))
+        authUser: JSON.parse(localStorage.getItem('authUser')),
       };
     }
 
     componentDidMount() {
-      console.log("storage: ", this.state.authUser);
       this.listener = this.props.firebase.auth.onAuthStateChanged(authUser => {
         if (authUser) {
-          console.log("listen: ", authUser);
-          localStorage.setItem("authUser", JSON.stringify(authUser));
-          this.setState({ authUser });
+          this.props.firebase.auth.currentUser.getIdToken().then(idToken => {
+            axios.post('/verify', { idToken: idToken }).then(response => {
```

Comment:
Should be the full URL here

Reply:
I set the base url in index.js so I don't have to write the full url in subsequent axios calls